src/containers/Callers/CallerDetailPanel.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Button, Col, message, Row, Skeleton, Typography } from 'antd';
 import EditCallerForm from './EditCallerForm';
+import styles from "./CallerDetailPanel.module.css";
```

Comment:
Right now the project is a mix of css modules and styled components using emotion. That's because when I set up the project, CSS Modules was what I was familiar with. A front-end developer who contributed to the project introduced styled components. Any opinions on which to settle on? I have enjoyed styled components because it has helped me break down units of HTML into more defined components, and also the ease of defining a styled component in the js file has helped me avoid the lazy habit of defining one-off inline styles rather than custom classes in the separate file.

Reply:
Hmm, I haven't really used emotion before, but I've just read up on it a little, and I'm familiar with the css-in-js concept.  What I'm actually most used to is plain old css, though, so I don't really know enough about either emotion / styled components or CSS Modules yet to have an informed opinion :shrug:

CSS Modules seems more familiar to me, but familiar doesn't always mean better!  I tend to err on the side of fewer dependencies, maybe, but if you like styled components, then there you go :)  I can give them a try next time I'm in the frontend code.